src/online2bin/online2-tcp-nnet3-decode-faster.cc

```diff
@@ -145,6 +160,8 @@ int main(int argc, char *argv[]) {
                 "Number of seconds of timout for TCP audio data to appear on the stream. Use -1 for blocking.");
     po.Register("port-num", &port_num,
                 "Port number the server will listen on.");
+    po.Register("produce-time", &produce_time,
```

Comment:
Just a suggestion: I think "--add-timestamps" or "--timestamps" is a better choice. "produce-time" seems a bit confusing to me.